ibis/backends/tests/test_json.py

```diff
@@ -48,6 +56,14 @@ def test_json_getitem(json_t, expr_fn, expected):
 @pytest.mark.notyet(
     ["pyspark", "trino"], reason="should work but doesn't deserialize JSON"
 )
+@pytest.mark.notimpl(
+    ["flink"],
+    raises=OperationNotDefinedError,
+    reason=(
+        "No translation rule for <class 'ibis.expr.operations.json.ToJSONMap'"
+        "Flink does not support ToJSONMap"
+    ),
+)
```

Comment:
if the upstream doesn't support it, this should be marked as a `notyet`

Reply:
@mfatihaktas I played around with this a bit on Friday, and while I can figure out how to check whether something is a JSON object, I wasn't able to find a way to actually construct a map from a string JSON object (even though it sounds like it should be doable...). I'm happy to go with `not yet`, as @gforsyth suggests, unless you know otherwise.